src/app/models/form.server.model.ts

```diff
@@ -857,6 +857,27 @@ const compileFormModel = (db: Mongoose): IFormModel => {
       .exec()
   }
 
+  FormSchema.statics.updateMsgSrvcName = async function (
+    formId: string,
+    msgSrvcName: string,
+  ) {
+    const form = await this.findById(formId).exec()
+
+    if (!form) return null
+
+    form.msgSrvcName = msgSrvcName
+    return form
+  }
+
+  FormSchema.statics.deleteMsgSrvcName = async function (formId: string) {
+    const form = await this.findById(formId).exec()
+
+    if (!form) return null
+
+    form.msgSrvcName = undefined
+    return form
+  }
+
```

Comment:
do you think this should be an document instance method instead of a model static method? This may mean we do not "double search" the db and can act directly on the document we presumably have already found (when identifying whether the user has permissions), and can call .save directly